testdata/default/cheats/RandomUint.t.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.18;
+
+import "ds-test/test.sol";
+import "cheats/Vm.sol";
+
+contract RandomUint is DSTest {
```

Comment:
We should have a test that `randomUint(uint256 min, uint256 max)` is always between min and max

Reply:
yep, didn't properly check if we have one

ptal @yash-atreya